report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
+
+\subsection{Motivating Example: NASA's Mars Missions}
+NASA's exploration of Mars, beginning with the Viking missions in the 1970s, has progressively deepened our understanding of Mars\cite{marsnasagov_vikings}. The \gls{msl} mission, which landed the Curiosity rover in Gale Crater in 2012, represents a pivotal step in this journey. Curiosity is equipped with the \gls{chemcam} instrument, a tool that uses \gls{libs} to analyze the chemical composition of Martian rocks and soils directly and non-invasively\cite{chemcamNasaWebsite}.
 
-\subsection{Problem Formulation}
-% repeats intro
-% Predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including high dimensionality, non-linearity, multicollinearity, and the phenomenon known as matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}.
-% Furthermore, the high cost of data collection often results in small datasets, complicating the task of building accurate and robust models.
+\gls{libs} is particularly suitable for the Martian environment because of its ability to perform rapid chemical analyzes remotely, creating a plasma that can be spectrally analyzed to determine the elemental composition of the vaporized material. This capability is crucial because it allows scientists to quickly and efficiently assess the geochemistry of multiple sites without physically moving the rover, thus conservatively managing the rover's limited energy and resources. The mission's focus has been on assessing past habitability, and the data gathered by \gls{chemcam} has been instrumental in identifying environments that could have supported life\cite{chemcamNasaWebsite,curiosityNasaWebsite}.
 
-Our problem definition builds upon the definitions provided in \citet{p9_paper}.
+The task of quantifying the oxides in Martian rock and soil samples begins with the \gls{libs} spectral data collected by Curiosity. This data comprises high-dimensional spectra with thousands of potential features, each corresponding to a specific element's emission lines. The computational challenge lies in accurately interpreting these complex data sets to deduce the concentrations of various elements, especially major oxides like iron, magnesium, and silicon, which are crucial for understanding Martian geology.
```

Comment:
"... This data comprises high-dimensional spectra with thousands of potential features, each corresponding to a specific element's emission lines."

are they potential features? Would it not be more correct to just call them features?

Reply:
Potential features because you preprocess, modify, and otherwise mask/remove various aspects of the data & even entire dimensions. So using the more relaxed 'potential' is appropriate IMO.

Reply:
Alright. I buy this argument. Resolved.